ui/components/component-library/text/README.mdx

```diff
@@ -0,0 +1,308 @@
+```
+
+### Color
+
+Use the `color` prop and the `TEXT_COLORS` object from `./ui/helpers/constants/design-system.js` to change the color of the `Text` component.
+
+
+<Canvas>
+  <Story id="ui-components-component-library-text-text-stories-js--color" />
+</Canvas>
+
+```jsx
+// If importing from ui/components/app/[YOUR_COMPONENT]/ directory
+import { Text } from '../../ui/component-library/text';
+import { TEXT_COLORS, BACKGROUND_COLORS } from '../../../helpers/constants/design-system';
+
+<Text color={TEXT_COLORS.TEXT_DEFAULT}>
+  text-default
+</Text>
+<Text color={TEXT_COLORS.TEXT_ALTERNATIVE}>
+  text-alternative
+</Text>
+<Text color={TEXT_COLORS.TEXT_MUTED}>
+  text-muted
+</Text>
+<Text color={TEXT_COLORS.OVERLAY_INVERSE} backgroundColor:{BACKGROUND_COLORS.OVERLAY_DEFAULT}>
+  overlay-inverse
+</Text>
+<Text color={TEXT_COLORS.PRIMARY_DEFAULT}>
+  primary-default
+</Text>
+<Text color={TEXT_COLORS.PRIMARY_INVERSE} backgroundColor:{BACKGROUND_COLORS.PRIMARY_DEFAULT}>
+  primary-inverse
+</Text>
+<Text color={TEXT_COLORS.ERROR_DEFAULT}>
+  error-default
+</Text>
+<Text color={TEXT_COLORS.ERROR_INVERSE} backgroundColor:{BACKGROUND_COLORS.ERROR_DEFAULT}>
+  error-inverse
+</Text>
+<Text color={TEXT_COLORS.SUCCESS_DEFAULT}>
+  success-default
+</Text>
+<Text color={TEXT_COLORS.SUCCESS_INVERSE} backgroundColor:{BACKGROUND_COLORS.SUCCESS_DEFAULT}>
+  success-inverse
+</Text>
+<Text color={TEXT_COLORS.WARNING_INVERSE} backgroundColor:{BACKGROUND_COLORS.WARNING_DEFAULT}>
+  warning-inverse
+</Text>
+<Text color={TEXT_COLORS.INFO_DEFAULT}>
+  info-default
+</Text>
+<Text color={TEXT_COLORS.INFO_INVERSE} backgroundColor:{BACKGROUND_COLORS.INFO_DEFAULT}>
+  info-inverse
+</Text>
+```
+
+### Font Weight
+
+Use the `fontWeight` prop and the `FONT_WEIGHT` object from `./ui/helpers/constants/design-system.js` to change the font weight of the `Text` component. There are 3 font weights:
+
+- `FONT_WEIGHT.NORMAL` = `normal` || `400`
+- `FONT_WEIGHT.MEDIUM` = `medium` || `500`
+- `FONT_WEIGHT.BOLD` = `bold` || `700`
+
+<Canvas>
+  <Story id="ui-components-component-library-text-text-stories-js--font-weight" />
+</Canvas>
+
+```jsx
+// If importing from ui/components/app/[YOUR_COMPONENT]/ directory
+import { Text } from '../../ui/component-library/text';
+import { FONT_WEIGHT } from '../../../helpers/constants/design-system';
+
+<Text fontWeight={FONT_WEIGHT.NORMAL}>
+  normal
+</Text>
+<Text fontWeight={FONT_WEIGHT.MEDIUM}>
+  medium
+</Text>
+<Text fontWeight={FONT_WEIGHT.BOLD}>
+  bold
+</Text>
+```
+
+### Font Style
+
+Use the `fontStyle` prop and the `FONT_STYLE` object from `./ui/helpers/constants/design-system.js` to change the font style of the `Text` component. There are 2 font styles:
+
+- `FONT_STYLE.NORMAL`
+- `FONT_STYLE.ITALIC`
+
+<Canvas>
+  <Story id="ui-components-component-library-text-text-stories-js--font-style" />
+</Canvas>
+
+```jsx
+// If importing from ui/components/app/[YOUR_COMPONENT]/ directory
```

Comment:
Do you think these comments add any value? We should either include them with all of our component documentation or not at all I'm thinking easier to remove WDYT?